kube-derive/src/lib.rs

```diff
@@ -111,6 +111,22 @@ mod custom_resource;
 /// Adding `#[kube(derive = "PartialEq")]` is required if you want your generated
 /// top level type to be able to `#[derive(PartialEq)]`
 ///
+/// ### `#[kube(schema_mode = "mode")]`
+/// Defines whether the `JsonSchema` of the top level generated type should be used when generating a `CustomResourceDefinition`.
+///
+/// Legal values:
+/// - `"derived"`: A `JsonSchema` implementation is automatically derived
+/// - `"custom"`: `JsonSchema` is not derived, but used when creating the `CustomResource` object
```

Comment:
How would one use `custom`? The manual schema override  uses disabled.

Reply:
Added an example that covers this case

Reply:
```suggestion
/// - `"custom"`: `JsonSchema` is not derived, but used when creating the `CustomResourceDefinition` object
```